react/src/pages/datasets/AnalysisRunnerDialog.tsx

```diff
@@ -19,7 +19,7 @@ import { annotations } from '../analysis/AnalysisInfoDialog';
 import ChipStrip from '../analysis/ChipStrip';
 
 interface AnalysisRunnerDialogProp {
-    participants: Participant[],
+    participants: Dataset[],
```

Comment:
The participants prop should probably be renamed to 'datasets' in this component